catboost/python-package/catboost/_catboost.pyx

```diff
@@ -5593,5 +5597,16 @@ cpdef compute_training_options(dict options, DataMetaInfo train_meta_info, DataM
     return loads(to_native_str(WriteTJsonValue(trainingOptions)))
 
 
+cpdef _get_onnx_model(model, export_parameters):
+    if not model._is_oblivious():
+        raise CatBoostError(
+            "ONNX-ML export is available only for models on oblivious trees ")
+
+    cpdef TString result = ConvertTreeToOnnxProto(
+        dereference((<_CatBoost>model).__model),
+        to_arcadia_string(export_parameters),
+    )
+    return bytes(result)
```

Comment:
Cython uses strlen to determine length of string before converting it to bytes, and if serialized model could contain zero char, it would cut result. To properly overcome that problem i would suggest to use explicit lenght specification:
```python
cdef const char* result_ptr = result.c_str()
cdef size_t result_len = result.size()
return bytes(result_ptr[:result_len])
```